armi/bookkeeping/db/database3.py

```diff
@@ -568,6 +569,46 @@ def open(self):
         self.h5db.attrs["armiLocation"] = os.path.dirname(armi.ROOT)
         self.h5db.attrs["startTime"] = armi.START_TIME
         self.h5db.attrs["machines"] = numpy.array(armi.MPI_NODENAMES).astype("S")
+        # store platform data
+        platform_data = uname()
+        self.h5db.attrs["platform"] = platform_data[0]
+        self.h5db.attrs["hostname"] = platform_data[1]
+        self.h5db.attrs["platformRelease"] = platform_data[2]
+        self.h5db.attrs["platformVersion"] = platform_data[3]
+        self.h5db.attrs["platformArch"] = platform_data[4]
```

Comment:
(piping in because I was tagged in the PR) this is great! Just learned about this package through this pull request.

One thing that might help readability would be to use the attributes provided via the [`uname` `namedtuple`](https://docs.python.org/3/library/platform.html#platform.uname), e.g.,
```python
platform = platform_data.system
hostname = platform_data.node
...
```

Reply:
Oh, it's a namedtuple.  When I printed it out, I thought it was just a regular tuple.  Nice catch, I'll make the change.

Reply:
Cool, I made the change.